web/src/app/common/InvalidUser/InvalidUserComponent.js

```diff
@@ -10,13 +11,24 @@ class InvalidUserComponent extends Component {
   render() {
     return (
       <div className={styles.missingContainer}>
-        <PageHeader>Oops! User not found</PageHeader>
+        <PageHeader>{this.props.title}</PageHeader>
         <div>
-          You can try finding them in the <LinkContainer exact to="/about"><Link to='/about'>streamer directory</Link></LinkContainer>
+          {this.props.message}
         </div>
       </div>
     );
   }
 }
 
+InvalidUserComponent.defaultProps = {
+  title: 'User not found',
+  message: 'We couldn\'t find this user. They may not have signed up for Bounty streamer yet.'
```

Comment:
Isnt it guaranteed they havent signed up for streamer contracts? Why it the wording 'may not have signed up'?

Reply:
Your right, the only other option is if the user doesn't exist on twitch